### DISCUSSION TITLE
Foundry is giving error

### DISCUSSION THREAD

**Discussion by ShubhSensei** [Cyfrin/foundry-full-course-cu#233] [Q&A] [answered]
I was about to start lesson 9 but somehow the whole `forge` commands stop working & I don't know why. I tried to reinstall it by this cmd `curl -L https://foundry.paradigm.xyz | bash` and now it's giving me the below error. 

<img width="855" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/91625601/b1ef20ef-7888-4003-875a-62f53aedbc54">

**Comment by ShubhSensei** [comment]
Also, when I try to run `forge`, it says this 
<img width="928" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/91625601/1ad1cf48-091f-47d5-a589-5ad6df9fa826">

**Comment by ghost** [comment]
Hello @SensationalShubham, check discussion [#206](https://github.com/Cyfrin/foundry-full-course-f23/discussions/206) to see if it helps you.

**Comment by IAm0xAxe** [comment]
Hey @SensationalShubham, My Solution to this problem:

- Open the hosts file and make changes to the "Github source". If you don't know how to solve it, you can ask Google/AI how to open the hosts file and how to change the "Github source". Similar to: 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/33568293/d8fafcc0-8622-43c7-ad0b-9d350b9b44d1)

- You can also ask the questions in the picture to the AI by using the "How to ask questions to the AI" as mentioned in the course content of the teacher (xxx).

I hope my suggestion will help you to solve this problem.

**Comment by ShubhSensei** [comment, marked as answer]
@pacelliv Thanx for the Reference bro. Problem Solved. But not by their suggested VPN & by another VPN [Proton](https://protonvpn.com/pricing). 
Guys, who so ever is facing this problem, just download this VPN & click on `Get Proton Free`. Once connected to another region, try again the installation process & it'll work. 👍
